rs_client.py

```diff
@@ -1186,6 +1187,20 @@ def _rs_union_accessor(typeobj, field):
         _r('}')
 
 
+    elif field.type.is_container:
+        _r('pub fn %s(&self) -> %s {', field.rs_field_name, field.rs_field_type)
+        with _r.indent_block():
+            _r('unsafe {')
+            with _r.indent_block():
+                if hasattr(field.type, 'struct_ptr') and field.type.struct_ptr:
+                    _r('std::mem::transmute(self)')
+                else:
```

Comment:
Is this casting a reference of `self` to a `StructPtr`? Does it bind lifetime of the `StructPtr` to the lifetime of `self`?